pkg/ocm/validations/getRole.go

```diff
@@ -0,0 +1,25 @@
+package validations
+
+import (
+	"net/url"
+	"strings"
+
+	errors "github.com/zgalor/weberr"
+)
+
+func ValidateIssuerUrlMatchesAssumePolicyDocument(
```

Comment:
This function doesn't match the file name, maybe we should for a more generic naming for the file?